pkgloader-recommended.sty

```diff
@@ -172,3 +172,23 @@
 %    \end{macrocode}
 %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+%  Author of selnolig at a comment to question
+%  <https://tex.stackexchange.com/q/419218/9075>
+%  on tex.stackexchange
+%
+%  > As a general rule, selnolig should be loaded after babel.
+%
+%    \begin{macrocode}
+\Load {babel} before  {selnolig}
+              if loaded
+              because {the author states that t it can matter
+                       in other circumstances}
```

Comment:
Stray `t` there. Also, please make the description more self-contained. What 'other' circumstances?

Reply:
The author literally said so at https://tex.stackexchange.com/questions/419218/how-to-use-selnolig-package-together-with-default-lualalatex-font#comment1049043_419218:

> Off-topic: As a general rule, selnolig should be loaded after babel. The load ordering doesn't affect the issue described here, but it can matter in other circumstances

@micoloretan Can you give us some more insights here? I assume @mhelvens wants to maintain a very good documentation of reasons for package ordering here.

Reply:
It doesn't have to be super-specific, but it shouldn't say "other circumstances", which is a phrase that only has meaning in that particular stackexchange thread.

How about:

```
              because {it was recommended by the selnolig author}
              % https://tex.stackexchange.com/questions/419218/how-to-use-selnolig-package-together-with-default-lualalatex-font#comment1049043_419218
```

Putting a relevant URL below the rule seems like a good practice.